fbtransfer/facebook-for-woocommerce
# Register WooCommerce Navigation items (updated)

I decided to take over the initial PR (#1698) to register WooCommerce nav items as there were merge conflicts and 2 newly added nav items that needed to be added. I had to open a new PR because the old one was from a fork branch.

Additionally I have hidden the existing tabbed navigation when the new Woo nav is active.

#### Screencast: with Woo nav enabled

![](https://d.pr/i/ZUEWIK+)

#### Screencast: With Woo Nav disabled

![](https://d.pr/i/knXm9I+)


## Testing instructions 

1. Clone and install/activate https://github.com/woocommerce/woocommerce-admin
1. Enable the navigation feature at `wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
1. Check that the "Facebook" extension is registered and menu items work as expected.
